Source/Core/Constants.swift

```diff
@@ -23,3 +23,43 @@
         }
     }
 }
+
+extension Constants {
+    enum Style {
+        enum BillingForm {
+            enum CancelButton: Double {
+                case height = 44
+                case width = 53
+                case fontSize = 17
+            }
+
+            enum DoneButton: Double {
+                case height = 44
+                case width = 53
+                case fontSize = 17
+            }
+
+            enum HeaderTitle: Double {
+                case fontSize = 24
+            }
+
+            enum InputErrorLabel: Double {
+                case height = 18
+            }
+
+            enum InputHintLabel: Double {
+                case fontSize = 13
+            }
+
+            enum InputTextField: Double {
+                case height = 56
+                case width = 335.0
```

Comment:
Why do we want to hard code sizes? Mobile devices come in different screen sizes and display types, why do we want to hardcode any sizes?

Reply:
For now, dynamic size is not supported. As per Patrick, the merchant will control size of every UI element. a new ticket will be added for this sizing.